pandas/core/frame.py

```diff
@@ -4168,14 +4168,14 @@ def dropna(self, axis=0, how='any', thresh=None, subset=None,
 
         Parameters
         ----------
-        axis : {0 or 'index', 1 or 'columns'}, or tuple/list thereof
+        axis : {0 or 'index', 1 or 'columns'}
             Determine if rows or columns which contain missing values are
             removed.
 
             * 0, or 'index' : Drop rows which contain missing values.
             * 1, or 'columns' : Drop columns which contain missing value.
 
-            Pass tuple or list to drop on multiple axes.
+            deprecated:: 0.23.0: Pass tuple or list to drop on multiple axes.
```

Comment:
The directive should start with a ".. " here, no?

Reply:
Fixed.